### ISSUE TITLE
Error when compiling quantized pytorch model with vai_c_xir: [UNILOG][Check Failed: shift_cut >= 0][/home/xbuild/conda-bld/xcompiler_1606385139459/work/src/slnode/DPUv2/SlNodeDptConv.cpp:55][XCOM_DATA_OUTRANGE][Data value is out of range!] 

### ISSUE THREAD

**Issue by wgshun** [Xilinx/Vitis-AI#964]
Hello,

I quantized a pytorch model and got an .xmodel. During compilation,  I get the following error:

```
VITIS_AI Compilation - Xilinx Inc.
**************************************************
[UNILOG][INFO] The compiler log will be dumped at "/tmp/vitis-ai-user/log/xcompiler-20220810-215919-99"
[UNILOG][INFO] Compile mode: dpu
[UNILOG][INFO] Debug mode: function
[UNILOG][INFO] Target architecture: DPUCZDX8G_CUSTOMIZED
[UNILOG][INFO] Graph name: Resnet, with op num: 604
[UNILOG][INFO] Begin to compile...
[UNILOG][Check Failed: shift_cut >= 0][/home/xbuild/conda-bld/xcompiler_1606385139459/work/src/slnode/DPUv2/SlNodeDptConv.cpp:55][XCOM_DATA_OUTRANGE][Data value is out of range!] 
*** Check failure stack trace: ***
Aborted (core dumped)
```

Has anyone had the same error or knows what might be causing this error message? 

I use Vitis-AI Docker vitis-ai-cpu:1.3.411.

Hope you can help me.
Thank you!

Here is the quantized xmodel:
链接：https://pan.baidu.com/s/1yu4aLBg_wIKxxHuFEvMaSA 
提取码：6m28

**Comment by wgshun** [comment]
Hi.

**Comment by niuxjxlnx** [comment]
Hi, @wgshun 

I noticed there are lots of scale layers converted from BatchNorm layer in your model, this is the root cause leads to deployment problem, which will make CONV layers feature maps not quantization friendly, deployment failure or not-acceptable performance of final deployment.

Suggest moving the BatchNorm layers to the output position of CONV layers, then they will be merged info CONV layers and will get reasonable deployment result.
